generators/client/templates/angular/src/main/webapp/app/account/password/password.component.html.ejs

```diff
@@ -32,63 +32,62 @@
                 The password and its confirmation do not match!
             </div>
 
-            <form name="form" role="form" (ngSubmit)="changePassword()" #passwordForm="ngForm">
+            <form name="form" role="form" (ngSubmit)="changePassword()" [formGroup]="passwordForm">
 
                 <div class="form-group">
                     <label class="form-control-label" for="currentPassword" jhiTranslate="global.form.currentpassword">Current password</label>
-                    <input type="password" class="form-control" id="currentPassword" name="currentPassword" #currentPasswordInput="ngModel"
+                    <input type="password" class="form-control" id="currentPassword" name="currentPassword"
                            placeholder="{{'global.form.currentpassword.placeholder' | translate}}"
-                           [(ngModel)]="currentPassword" required>
-                    <div *ngIf="currentPasswordInput.dirty && currentPasswordInput.invalid">
+                           formControlName="currentPassword">
+                    <div *ngIf="passwordForm.get(['currentPassword']).invalid && (passwordForm.get(['currentPassword']).dirty || passwordForm.get(['currentPassword']).touched)" >
                         <small class="form-text text-danger"
-                               *ngIf="currentPasswordInput.errors.required" jhiTranslate="global.messages.validate.newpassword.required">
+                               *ngIf="passwordForm.get(['currentPassword']).errors.required" jhiTranslate="global.messages.validate.newpassword.required">
```

Comment:
use the string parameter instead of array one: `.get('currentPassword')`

Reply:
Done for each fields!